api/edge_api/identities/views.py

```diff
@@ -140,6 +143,20 @@ def update_traits(self, request, *args, **kwargs):
         return Response(data, status=status.HTTP_200_OK)
 
 
+@method_decorator(
```

Comment:
I think you can simplify this quite a lot: 

 1. Just add the `swagger_auto_schema` decorator directly to the `list` method (since we're already overriding it anyway). 
 2. Use the `query_serializer` argument instead of the `manual_parameters`

So you should end up with something like: 

```python
@swagger_auto_schema(query_serializer=EdgeIdentityFsQueryparamSerializer())
def list(self, request, *args, **kwargs):
    ...
```

Reply:
Neat!